e2e/testsuite/testsuite.go

```diff
@@ -103,6 +141,111 @@ func (s *E2ETestSuite) configureGenesisDebugExport() {
 	t.Setenv("EXPORT_GENESIS_CHAIN", genesisChainName)
 }
 
+// SetupChains creates the chains for the test suite, and also a relayer that is wired up to establish
+// connections and channels between the chains.
+func (s *E2ETestSuite) SetupChains(ctx context.Context, channelOptionsModifier ChainOptionModifier, chainSpecOpts ...ChainOptionConfiguration) {
+	s.T().Logf("Setting up chains: %s", s.T().Name())
+	s.initState()
+	s.configureGenesisDebugExport()
+
+	chainOptions := DefaultChainOptions()
+	for _, opt := range chainSpecOpts {
+		opt(&chainOptions)
+	}
+
+	s.chains = s.createChains(chainOptions)
+
+	// TODO: we need to create a relayer for each test that will run
+	// having a single relayer for all tests will cause issues when running tests in parallel.
```

Comment:
Assuming this TODO is directly related to the one on`GetRelayer` below on this file. 

What issues get caused by having a single relayer for tests running in parallel? 🤔 Just curious about the specifics 

We can create an issue to follow up on this too ofc! 👍🏻